cpp-ch/local-engine/Common/CHUtil.cpp

```diff
@@ -674,7 +674,7 @@ void BackendInitializerUtil::initContexts(DB::Context::ConfigurationPtr config)
 
         auto getDefaultPath = [config] -> auto
         {
-            bool use_current_directory_as_tmp = config->getBool("use_current_directory_as_tmp", false);
```

Comment:
Usually，in yarn mode, current directory is at the data disk, but /tmp/ will use OS default disk. So use_current_directory_as_tmp setting to true is better.I can change it back if i misstake something.

Reply:
It will introduce backward incompatiability for other uses who have already use gluten. Please keep it as it was, and explicitly set it to false in spark conf if you need.

Reply:
fixed